### PR TITLE
Validate Partial

### PR DESCRIPTION
**Pull request by Fluidbyte**

### CODE REVIEW

File: src/rules.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ const rules = {
    * @memberof rules
    * @param {Object} def The rule definition object
    * @param {*} data The data (value) to validate
+   * @param {Object} opts Specific options for validation process
```
Comment (TomFrost) [inline src/rules.js]: Should change to mark this as optional/list defaults in the docs:

```
@param {Object} [opts={partial: false}] Specific options for validation process
```

**Comment by TomFrost** [comment]
:accept:

**Comment by TomFrost** [comment]
(Err rather, :accept: after the doc generator is re-run)

**Comment by Fluidbyte** [comment]
right, of course.

**Pull request merged**